eiger_io/pims_reader.py

```diff
@@ -17,12 +17,12 @@
 
 class EigerImages(FramesSequence):
     pattern = re.compile('(.*)master.*')    
-    def __init__(self, master_filepath):
```

Comment:
@ericdill Changing the signature of the `__init__` is basically defining a new SPEC. Unless I'm confused, this handler will not work on any data saved according to the `'AD_EIGER'` spec. Would you like to create a new spec? Have you already done this at xf11id?